src/data/stig_multipliers.ts

```diff
@@ -215,6 +215,38 @@ const SUPPORT_STIG_MULTIPLIERS = Object.freeze({
       }),
     ],
   },
+  "Shattered Swords": {
+    B: [
+      new Multiplier({
+        active: true,
+        name: "Empyrea Phoenix B: Combo atk boosts team Crit DMG for 15s",
```

Comment:
Let's use the stig set name here, ex. "Shattered Swords B:"

For Rebel blades pieces, we use the individual piece name in the description only because the piece name people click on is "Kiana T", "Mei M", etc.

Reply:
Changed from `Empyrea Phoenix B` to `Shattered Swords B` in latest commit.